server/routerlicious/packages/services-core/src/mongoCheckpointRepository.ts

```diff
@@ -62,6 +62,19 @@ export class MongoCheckpointRepository implements ICheckpointRepository {
 		documentId: string,
 		tenantId: string,
 	): { _id: string; documentId: string } & any {
+		const isError = !documentId || !tenantId;
+
+		if (isError) {
+			const error = new Error(`Cannot create filter due to missing parameter`);
+			Lumberjack.error(
+				"Missing parameter when writing checkpoint.",
+				{
+					...getLumberBaseProperties(documentId, tenantId),
```

Comment:
with just base properties, we can just do 
`Lumberjack.error("Missing parameter when writing checkpoint.", getLumberBaseProperties(documentId, tenantId), error);`, right? No need to do `{...}` thing